frontend/mr-admin-flate/src/components/avtaler/OpprettAvtaleContainer.module.scss

```diff
@@ -1,13 +1,26 @@
-.flex {
+.container {
   display: flex;
-  align-items: center;
-  justify-content: space-between;
+  flex-direction: column;
+  align-items: space-between;
+  height: 100%;
```

Comment:
Kan slenge på en `flex-wrap: wrap` her. Så får vi en kolonne når skjermen blir for liten